spec/models/target_spec.rb

```diff
@@ -34,4 +34,77 @@
     it { is_expected.to belong_to(:user) }
     it { is_expected.to belong_to(:topic) }
   end
+
+  describe 'functions' do
+    let!(:second_user)     { create(:user) }
+    let!(:topic)           { create(:topic) }
+    let!(:target_params)   { attributes_for(:target) }
+    let!(:second_target) do
+      create(:target, user: second_user, topic: topic,
+                      radius: 0,
+                      latitude: target_params[:latitude],
+                      longitude: target_params[:longitude])
+    end
+
+    describe '.check_matching_targets' do
+      context 'when targets match' do
+        subject do
+          create(:target, user: user, topic: topic,
+                          latitude: target_params[:latitude],
+                          longitude: target_params[:longitude])
+        end
+
+        it 'calls for notification job' do
+          ActiveJob::Base.queue_adapter = :test
```

Comment:
isn't this by default in tests?

Reply:
🤷‍♀ idk,
it makes use of it [here](https://relishapp.com/rspec/rspec-rails/docs/job-specs/job-spec)